nym-vpn-app/src-tauri/src/sentry.rs

```diff
@@ -1,11 +1,31 @@
-use sentry::{ClientInitGuard, User};
-use std::{borrow::Cow, time::Duration};
-use tracing::{info, instrument, warn};
+use sentry::{ClientInitGuard, Level, User};
+use std::{
+    borrow::Cow,
+    collections::HashSet,
+    sync::{Arc, OnceLock},
+    time::Duration,
+};
+use tracing::{info, warn};
 
 use crate::env::APP_SENTRY_DSN;
 use crate::sys::OsInfo;
 
-#[instrument(skip_all)]
+static EXCLUDED_ERRORS: OnceLock<HashSet<&'static str>> = OnceLock::new();
+
+fn get_excluded_errors() -> &'static HashSet<&'static str> {
+    EXCLUDED_ERRORS.get_or_init(|| {
+        HashSet::from([
+            "invalid mnemonic",
+            "no device stored",
+            "no account stored",
+            "ac is offline",
+            "account already exists",
+            "maxdevicesreached",
+            "subscriptionexpired",
+        ])
+    })
+}
```

Comment:
To streamline this pattern switch to `once_cell`' Lazy (this crate is already in use in tauri)

```
static EXCLUDED_ERRORS: Lazy<HashSet<&'static str>> = Lazy::new(|| {
        HashSet::from([...]);
});
```